readme.txt

```diff
@@ -1,9 +1,9 @@
 === Plugin Name ===
 Contributors: boldgrid, fredericktownes, maxicusc, gidomanders, bwmarkle, harryjackson1221, joemoto, vmarko, jacobd91
 Tags: seo, cache, CDN, pagespeed, caching, performance, compression, optimize, cloudflare, nginx, apache, varnish, redis, aws, amazon web services, s3, cloudfront, azure
-Requires at least: 3.8
+Requires at least: 5.3
```

Comment:
I'm saddened after reporting an issue https://github.com/W3EDGE/w3-total-cache/issues/547
That my version of WP is no longer supported. WP 4.9.2